<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta name="apple-mobile-web-app-capable" content="yes">

<style>
@import url(CSS/stack.css);
@import url(CSS/d3-tip.css);
@import url(CSS/figures.css);
@import url(CSS/kbroman_talk.css);
/* comment following line to get rid of notes; uncommented to show notes */
/* @import url(CSS/kbroman_presentation.css); */
</style>

<script charset="utf-8" type="text/javascript" src="JS/d3.js"></script>
<script type="text/javascript" src="JS/d3.tip.js"></script>




<title>QTL mapping for phenotypes measured over time</title>
</head>
<body class="overview">

<!-- for the movie : put in head -->
<!--
<script src="Movie/scripts/prototype.js" language="JavaScript" type="text/javascript"></script>
<script src="Movie/scripts/qtp_poster.js" language="JavaScript" type="text/javascript"></script>
<link href="Movie/stylesheets/qtp_poster.css" rel="StyleSheet" type="text/css" />
-->




<section class="titlepage">
<h2 class="highlight">QTL mapping for phenotypes measured over time</h2>

<h4><a href="http://www.biostat.wisc.edu/~kbroman">Karl Broman</a></h4>
<p class="nopadding">Biostatistics &amp; Medical Informatics, University of Wisconsin
&ndash; Madison</p>

<p class="nopadding"><code><a href="http://www.biostat.wisc.edu/~kbroman"
class="gray">http://www.biostat.wisc.edu/~kbroman</a></code></p>

<p class="codelink"><a href="presentation.html" class="gray">Remove stickies</a>

<aside>
Click on the link at the bottom-left to remove these
notes.<br><br>

These slides were tested for Safari on a Mac and might look terrible
in some browsers.  I have a lot more to learn about handling browser
differences.
</aside>

</section>



<section>

<h3>Inbred mice</h3>

<img style="position:fixed;bottom: 0px; left:0px; z-index: -1;" width="1280px" src="Figs/inbredmice.jpg" />

<aside>
This talk is for a symposium on plant breeding, so I should start by
emphasizing that I focus mostly on mouse genetics.<br><br>

The goals for QTL analysis in plant breeding and in mouse genetics can
be quite different, and so the appropriate methods can be quite
different.<br><br>

I think these are <a
href="http://jaxmice.jax.org/strain/000689.html">SWR mice</a>; the
photo is from <a
href="http://cals.ncsu.edu/genetics/index.php/people/david-threadgill/">David
Threadgill</a>.
</aside>

<p class="pg">2</p></section>






<section>

<h3>Human vs mouse</h3>

<img style="position:fixed;top: 300px; left:120px; z-index: -1;" height="500px" src="Figs/da-vinci-man.jpg" />

<img style="position:fixed;top: 300px; right:120px; z-index: -1;" height="500px" src="Figs/vitruvian_mouse.jpg" />

<aside>
Mice are not humans, but you can learn a great deal about human
biology and disease from mice.<br><br>

The figure on the right is from <a
href="http://www.daviddeen.com">David Deen</a>.
</aside>


<p class="pg">3</p></section>


<section>

<h3>Gravitropism</h3>

<div style="position:absolute;top:312px;left:420px;font-size:20px;">
<a href="Movie/gmovie.mov" rel="qtposter">
  <img src="Movie/initial_view.png" width="440" alt="Phenotype movie"/>
</a></div>

<aside>
This work was motivated by (and will be illustrated through) data from
a collaboration with Edgar Spalding and colleagues, on <a
href="http://en.wikipedia.org/wiki/Gravitropism">gravitropism</a> in
Arabidopsis.<br><br>

A seedling was sprouted, and rotated 90 degrees, and then the response
followed with a camera.<br><br>

Click on the image to see the movie.<br><br>

You&rsquo;ll be popped to a separate page; you&rsquo;ll need to navigate back.
</aside>

<p class="pg">4</p></section>

</section>



<section>

<h3>Recombinant inbred lines</h3>


<div align="center">
<img src="Figs/riself.png" />
</div>


<aside>
The data concern a set of 162 recombinant inbred lines (RIL).<br><br>

RIL are formed by crossing two inbred strains followed by repeated
inbreeding (in plants, generally by selfing).  The result is a set of
new inbred lines whose genomes are mosaics of the original two lines.
</aside>


<p class="pg">5</p></section>

</section>





<section>

<h3>Data</h3>


<div align="center">
<img src="Figs/geno_pheno.png" />
</div>


<aside>
The data consist of marker genotypes (AA vs BB) for each line at a set
of markers across the genome, and a phenotype (&ldquo;tip angle&rdquo;)
measured over time for each line.
</aside>


<p class="pg">6</p></section>

</section>



<section nopadding>

<h3 style="margin-top:10px;margin-left:-20px;">Phenotypes</h3>

<div id="phenofig" style="margin-top:-50px;margin-left:100px;"></div>

<aside>
The top panel is an image plot of the phenotype, measured over time,
for many lines.  The rows correspond to lines.
There is a color scale at the bottom right.<br><br>

Hover over a row to see the detailed time course below.<br><br>

Click on a row and the curve below will be retained as you move
away.  Click again (if you can get your pointer onto it) and it will
be removed.
</aside>


<p class="codelink">
<a href="Coffee/pheno.coffee" class="gray">code</a> |
<a href="Data/pheno.json" class="gray">data</a>


<script type="text/javascript" src="JS/pheno.js"></script>
<p class="pg">7</p></section>



<section>

<h3>LOD curves at 62 min</h3>

<div id="lod_onetime_fig" style="margin-left:-30px;"></div>

<aside>
LOD curves and QTL effect plot for a single time point.<br><br>

The LOD score at a genomic position is a measure of association
between genotype and phenotype: Is there evidence for a QTL in the
region?
</aside>


<p class="codelink">
<a href="Coffee/lod_onetime.coffee" class="gray">code</a> |
<a href="Data/ontime.json" class="gray">data</a>


<script type="text/javascript" src="JS/lod_onetime.js"></script>
<p class="pg">8</p></section>




<section>

<h3>LOD curves at 62 min</h3>

<div id="lod_onetime_random_fig" style="margin-left:-30px;"></div>

<aside>
This is an illustration of a permutation test: To assess whether a
given LOD score is sufficiently large to indicate evidence of a QTL,
we compare it to the distribution of the genome-wide maximum LOD score
under the null hypothesis of no QTL.<br><br>

The null distribution of the genome-wide maximum LOD is estimated by
permuting the phenotypes relative to the genotypes.<br><br>

Click the green button to shuffle the phenotypes and re-draw the LOD
curves.<br><br>

There&rsquo;s a hidden &ldquo;back&rdquo; button a bit to the right of
the green button.
</aside>


<p class="codelink">
<a href="Coffee/lod_onetime.coffee" class="gray">code</a> |
<a href="Data/ontime.json" class="gray">data</a>


<script type="text/javascript" src="JS/lod_onetime_random.js"></script>
<p class="pg">9</p></section>



<section>

<h3>Permutations</h3>


<div align="center">
<img src="Figs/perm_hist.png" />
</div>


<aside>
This is the distribution of the genome-wide maximum LOD score, across
100,000 permutations.<br><br>

You can treat the 95th percentile, 2.60, as a
significance thresold.<br><Br>

Alternatively, convert the peak LOD score into a p-value,
by taking the proportion of permutations with maximum LOD meeting or
exceeding that value.
</aside>


<p class="pg">10</p></section>

</section>






<section nopadding>

<h3>Single-QTL analyses</h3>


<div align="center">
<a href="alt_alltimes.html"><img src="Figs/alltimes_static.png" /></a>
</div>

<aside>
[This is a static shot; click on the image to see a
dynamic version.]<br><br>

The top-left plot is a &ldquo;heat map&rdquo; of the LOD scores
for each time point at each genomic position.  LOD scores are also
colored to indicate the sign of the QTL effect, with red indicating
that BB lines have larger phenotype values and blue indicating that AA lines
have larger phenotype values.  We consider only those (position, time)
pairs with LOD > 1.<br><br>

When you hover over a point in the top-left plot, the LOD curves for the
corresponding time are shown below, and the phenotype averages and
estimated QTL effect (across time) for the corresponding genomic
position are shown to the right.<br><br>

Click on a point to show pointwise confidence bands on the QTL
effect (&plusmn; 2 SE).  (We require a mouse click, as otherwise the graph
was painfully slow to refresh.)
</aside>


<p class="codelink">
<a href="Coffee/lod_alltimes.coffee" class="gray">code</a> |
<a href="Data/alltimes.json" class="gray">data</a>

<p class="pg">11</p></section>


<section>

<h3>Phenotypes measured over time</h3>

<ul class="tight" style="margin-top:75px;">
  <li>Consider each time point individually</li>

  <li>Combine the results from the individual analyses

      <p class="lowlight"><font class="small">SLOD(&lambda;) = ave<sub>t</sub> LOD<sub>t</sub>(&lambda;)<br>
                         MLOD(&lambda;) = max<sub>t</sub> LOD<sub>t</sub>(&lambda;)</font></p>

      </li>

  <li>Fit a model to each individual&rsquo;s curve; <br>treat the coefficients as phenotypes</li>
</ul>


<aside>
How to handle the multiple time points?<br><br>

The simplest thing is just to treat each time point as a separate
phenotype.  Or we could do that but that combine the results across
time points in some way.<br><br>

Alternatively, one could fit a model to each individual&rsquo;s curve and
treat the estimated coefficients as phenotypes.  There are a variety
of fancy versions of this.<br><br>

I&rsquo;m focusing on the simplest approach: treat each time point
individually, and then combine the results.
</aside>


<p class="pg">12</p></section>

</section>



<section>
<h3>Results<font class="lowlight">: SLOD & MLOD</font></h3>

<div align="center">
<img src="Figs/slod_mlod.png" />
</div>

<aside>
Results for single-QTL analysis with SLOD and MLOD.<br><br>

With SLOD, we infer that there are QTL on chr 1 and 4.  There is
nearly significant evidece for another QTL, on chr 5.<br><br>

With MLOD, we infer that there are QTL on chr 1, 3, and 4.
</aside>

<p class="pg">13</p></section>
</section>



<section>
<h3>Multiple-QTL models</h3>

<ul class="tight" style="margin-top:75px;">
  <li>Reduce residual variation &rarr; Increase power</li>

  <li>Separate linked QTL</li>

  <li>Identify interactions among QTL <font class="lowlight">(epistasis)</font></li>
</ul>

<aside>
There are three main reasons that one should consider multiple-QTL
models.<br><br>

First, by controlling for major loci, one can increase power to detect
more modest loci.<br><br>

Second, to determine whether there are two QTL on a chromosome,
one should fit a two-QTL model and compare it to the best
single-QTL model.<br><br>

Finally, the investigation of epistatic interactions requires the
consideration of multiple-QTL models.
</aside>

<p class="pg">14</p></section>
</section>



<section>
<h3>Penalized LOD</h3>

<p class="gray" style="margin-left:125px;margin-top:75px;">Among additive QTL models, maximize</p>

<div align="center" style="margin-top:30px;"><p class="highlight">pLOD(&gamma;) = LOD(&gamma;) &ndash; T &times; |&gamma;|
</div>

<p class="lowlight" style="margin-left:600px;margin-top:150px;">
&gamma; = a multiple-QTL model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
|&gamma;| = no. QTL in the model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
T = significance threshold</p>


<p class="codelink">
<a class="gray" href="http://www.biostat.wisc.edu/~kbroman/publications/jrssb.pdf">KW Broman, TP Speed (2002) J Roy Stat Soc B 64:641-656</a>
</p>

<aside>
The key issue in choosing among multiple-QTL models is the criterion
for comparing models.<br><br>

If you add a QTL to a model, the LOD score will increase; how much of
an increase in LOD should be required to select the additonal
QTL?<br><br>

We seek to balance model fit with model complexity.<br><br>

In particular, we use as penalty the significance threshold from a
single-QTL scan, derived via a permutation test.
</aside>

<p class="pg">15</p></section>
</section>



<section>
<h3>Penalized LOD</h3>

<p class="gray" style="margin-left:125px;margin-top:75px;">Among additive QTL models, maximize</p>

<div align="center" style="margin-top:30px;"><p class="highlight">pLOD<sub>t</sub>(&gamma;) = LOD<sub>t</sub>(&gamma;) &ndash; T &times; |&gamma;|
</div>

<p class="lowlight" style="margin-left:600px;margin-top:150px;">
&gamma; = a multiple-QTL model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
|&gamma;| = no. QTL in the model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
T = significance threshold</p>


<p class="codelink">
<a class="gray" href="http://www.biostat.wisc.edu/~kbroman/publications/jrssb.pdf">KW Broman, TP Speed (2002) J Roy Stat Soc B 64:641-656</a>
</p>

<aside>
We can apply this approach to each time, individually, by just adding
&ldquo;t.&rdquo;
</aside>

<p class="pg">15</p></section>
</section>


<section>
<h3>Forward selection</h3>

<div align="center">
<img src="Figs/forwsel1.png" />
</div>

<aside>
Here we illustrate the forward selection process for tip angle at 62
min.<br><br>

The first step is just to perform a genome scan with a single-QTL
model.  We find a QTL on chromosome 4.
</aside>

<p class="pg">16</p></section>
</section>

<section>
<h3>Forward selection</h3>

<div align="center">
<img src="Figs/forwsel2.png" />
</div>

<aside>
In the second step of forward selection, we scan for an additional
QTL, conditional on the identified QTL on chr 4.<br><br>

We have evidence for a second QTL on chr 3.
</aside>

<p class="pg">16</p></section>
</section>

<section>
<h3>Forward selection</h3>

<div align="center">
<img src="Figs/forwsel3.png" />
</div>

<aside>
We then do another genome scan, conditional on the selected QTL on chr
3 and 4.<br><br>

The next-highest peak is on chr 1, but it doesn&rsquo;t meet the threshold,
so we stop.<br><br>

In practice, we use forward selection to a large model (say 10 QTL)
and then do the reverse (&ldquo;backward delection&rdquo;) back down
to the null model, and we select the model with maximum pLOD, among
all models considered.
</aside>

<p class="pg">16</p></section>
</section>

<section>
<h3>LOD profile</h3>

<div align="center">
<img src="Figs/lodprofile.png" />
</div>

<aside>
In this figure, we display the &ldquo;profile LOD curves&rdquo; for
the selected model (with QTL on chr 3 and 4).<br><br>

The LOD curve on chr 3 compares the two-QTL model (with QTL on chr 3
and 4, varying the location of the QTL on chr 3 but keeping the QTL on
chr 4 fixed at its peak location) to the single-QTL model (with just
the QTL on chr 4).<br><br>

Similarly, the LOD curve on chr 4 compares the two-QTL model to the
single-QTL model with just the QTL on chr 3.<br><br>

The curves on the other chromosome are for a comparison of a three-QTL
model (with a third QTL added to the model with QTL on chr 3 and 4) to
the two-QTL model (with just the QTL on chr 3 and 4).
</aside>

<p class="pg">17</p></section>
</section>



<section>
<h3>Multiple-QTL analyses</h3>

<div align="center">
<a href="alt_stepwise.html"><img src="Figs/stepwise_static.png" /></a>
</div>

<aside>
[This is a static shot; click on the image to see the
dynamic version.]<br><br>

This shows the results of multiple-QTL analysis with each time point
individually.<br><br>

When you hover over a time point in the top panel, selected QTL are
indicated, and the profile LOD curves are shown in the lower
panel.<br><br>

A key issue, with the model selection, is the discrete jumps between
models from time point to time point.
</aside>

<p class="codelink">
<a href="Coffee/stepwise.coffee" class="gray">code</a> |
<a href="Data/stepwise.json" class="gray">data</a>

<p class="pg">18</p></section>
</section>



<section>
<h3>Penalized LOD<font class="lowlight">, extended</font></h3>

<p class="gray" style="margin-left:125px;margin-top:75px;">Among additive QTL models, maximize</p>

<div align="center" style="margin-top:30px;">
<p class="highlight">pSLOD(&gamma;) = ave<sub>t</sub> LOD<sub>t</sub>(&gamma;) &ndash; T<sub>S</sub> &times; |&gamma;|</p>
<p class="highlight">pMLOD(&gamma;) = max<sub>t</sub> LOD<sub>t</sub>(&gamma;) &ndash; T<sub>M</sub> &times; |&gamma;|</p>
</div>


<p class="lowlight" style="margin-left:600px;margin-top:150px;">
&gamma; = a multiple-QTL model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
|&gamma;| = no. QTL in the model</p>

<p class="lowlight" style="margin-left:600px;margin-top:-20px;">
T = penalties</p>


<aside>
The penalized LOD score idea can be immediately extended to combine the
information across time points by just inserting SLOD or
MLOD in place of LOD<sub>t</sub>.
</aside>

<p class="pg">19</p></section>
</section>



<section>
<h3>Results<font class="lowlight">: SLOD</font></h3>

<div align="center">
<img src="Figs/slod_multiqtl.png" />
</div>

<aside>
Profile LOD curves and estimated QTL effects for the 3-QTL model selected
with the pSLOD criterion.<br><br>

Below are the estimated QTL effects across time.
</aside>

<p class="pg">20</p></section>
</section>



<section>
<h3>Summary</h3>

<ul class="tight" style="margin-top:75px;">
  <li>We have focused on the simplest approaches to QTL analysis with a
  phenotype measured over time.</li>
       <ul class="lowlight">
       <li>Consider each time point individually.</li>
       <li>Combine across time by taking the average or maximum
  LOD.</li>
     </ul>
  <li>We extended the approach for multiple-QTL models.</li>
  <li>Interactive graphs are useful and fun.</li>
</ul>

<aside>
It&rsquo;s always good to have a summary slide.
</aside>

<p class="pg">21</p></section>
</section>



<section>
<h3>Acknowledgments</h3>

<div style="font-size:30px;">

<table cellspacing=25 style="margin-left:150px;">
  <tr valign="top">
     <td><a href="http://www.botany.wisc.edu/spalding.htm" class="gray">Edgar Spalding</a></td>
     <td width=20></td>
     <td class="lowlight">Botany, UW-Madison</td>
     </tr>
  <tr><td><a href="http://www.botany.wisc.edu/moore.htm" class="gray">Candace Moore</a></td></tr>
  <tr><td><a href="http://www.botany.wisc.edu/johnson.htm" class="gray">Logan Johnson</a></td></tr>
  <tr height=75><td></td></tr>
  <tr valign="top">
     <td>Il-youp Kwak</td>
     <td></td>
     <td class="lowlight">Statistics, UW-Madison</td>
     </tr>
  <tr height=75><td></td></tr>
  <tr valign="top">
     <td colspan=3><font class="highlight">NIH</font><font class="lowlight">:
  R01 GM074244</font></td>
     </tr>
</table>
</div>

<aside>
This work is in collaboration with <a
href="http://www.botany.wisc.edu/spalding.htm">Edgar Spalding</a> and
colleagues, and my graduate student, Il-youp Kwak.<br><br>

The work was supported in part by a grant from the NIH.
</aside>

<p class="pg">22</p></section>





<script type="text/javascript" src="JS/stack.v0.js"></script>
</body>
</html>

